config/fortify.php

```diff
@@ -6,6 +6,7 @@
     'guard' => 'web',
     'passwords' => 'users',
     'username' => 'email',
+    'email_address' => 'email',
```

Comment:
Can we name this `email` to be consistent with other apps/packages in the eco system?

Reply:
I have made the requested changes.